style.css

```diff
@@ -75,13 +77,24 @@ h3 {
   margin-left: -5px;
   border-width: 5px;
   border-style: solid;
-  border-color: grey transparent transparent transparent; 
+  border-color: grey transparent transparent transparent;
 }
 
 
 .type {
-    height: 10px;
-    width: 10px;
+    height: 25px;
+    width: 66px;
+    margin-bottom: 4px;
+    border-radius: 4px;
+    border: 1px solid;
+    color: #ffff;
+    font-size: .75rem;
+    font-weight: normal;
+    font-family: helvetica, sans-serif;
```

Comment:
When you add a font, its always a good idea to add a back-up incase the computer doesn't have it like 'font-family: helvetica, sans-serif, Arial;'

Reply:
I didn't know about that, cool, updated!

Reply:
All good! 👍